src/05-objects-tasks.js

```diff
@@ -51,8 +54,11 @@ function getJSON(/* obj */) {
  *    const r = fromJSON(Circle.prototype, '{"radius":10}');
  *
  */
-function fromJSON(/* proto, json */) {
-  throw new Error('Not implemented');
+function fromJSON(proto, json) {
+  const jsonObj = JSON.parse(json);
+  Object.setPrototypeOf(jsonObj, proto);
```

Comment:
I think it will be better to use the "Object.create()" method for prototype setting.